packages/perspective-viewer-d3fc/src/js/charts/area.js

```diff
@@ -50,7 +50,8 @@ function areaChart(container, settings) {
     const zoomChart = zoomableChart()
         .chart(chart)
         .settings(settings)
-        .xScale(xAxis.scale);
+        .xScale(xAxis.scale)
+        .dateAxis(xAxis.type === AXIS_TYPES.time);

```

Comment:
The `zoomableChart` component is given the `xScale` object to use, so I was hoping there would be some way to determine whether that was a date scale without adding any further properties, which would avoid all these extra property setters.

e.g. Perhaps you could look at the scale's domain, or else look at the type of the scale object itself.